samples/subsys/fs/littlefs/boards/nucleo_h743zi.conf

```diff
@@ -19,3 +19,5 @@ CONFIG_FS_LITTLEFS_PROG_SIZE=32
 CONFIG_FS_LITTLEFS_READ_SIZE=32
 CONFIG_FS_LITTLEFS_LOOKAHEAD_SIZE=64
 CONFIG_FS_LITTLEFS_FC_HEAP_SIZE=8192
+
+CONFIG_NOCACHE_MEMORY=y
```

Comment:
Shouldn't this be `CONFIG_DCACHE=n`?

Reply:
No, this board with this test has been checked that works correctly.